include/SceneRenderer.h

```diff
@@ -107,7 +108,7 @@ class SceneRenderer : public Singleton<SceneRenderer>
 
     void      clearRadar(float opacity);
 
-    void      getGroundUV(const float p[2], float uv[2]) const;
+    void      getGroundUV(const glm::vec2 p, glm::vec2 uv) const;
```

Comment:
will this work? `uv` appears to be pass-by-value, but I suspect the method is trying to populate it.